extension-requests/script.js

```diff
@@ -227,7 +227,18 @@ const getExtensionColor = (deadline, createdTime) => {
   return 'orange-text';
 };
 
+const currentUserDetailsPromise = getSelfUser()
+  .then((response) => {
+    currentUserDetails = response;
+  })
+  .catch((error) => {
+    currentUserDetails = null;
```

Comment:
why are we not handling the error here?

Reply:
How do you think it should be handled?

Reply:
@fakhruddinkw Added toast for displaying error.